raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -654,4 +685,68 @@ describe('Raiden', () => {
       }),
     ).resolves.toEqual(channelOpenHash);
   });
+
+  test('suggestPartners', async () => {
+    const deps = makeDummyDependencies();
+    const raiden = new Raiden(
+      makeInitialState({ address, network, contractsInfo }, { tokens: { [token]: tokenNetwork } }),
+      deps,
+      combineRaidenEpics([initEpicMock]),
+      dummyReducer,
+    );
+    await raiden.start();
+    await raiden.monitorToken(token);
+
+    const suggestedPartners = raiden.suggestPartners(token);
+    await expect(suggestedPartners).resolves.toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          address: expect.toBeString(),
+          capacity: BigNumber.from(1),
+          centrality: 2,
+          score: 3,
+          uptime: 4,
+        }),
+      ]),
+    );
+  });
+
+  test('transferOnchainBalance', async () => {
+    const deps = makeDummyDependencies();
+    deps.signer = {
+      sendTransaction: jest.fn(async () => ({
+        wait: jest.fn(async () => ({
+          status: 1,
+        })),
+        hash: txHash,
+      })) as any,
+    } as any;
+
+    const raiden = new Raiden(
+      makeInitialState({ address, network, contractsInfo }),
+      deps,
+      combineRaidenEpics([initEpicMock]),
+      dummyReducer,
+    );
+    raiden.getBalance = jest.fn(async () => BigNumber.from(1_000_000)) as any;
+
+    await raiden.start();
+    const tx = raiden.transferOnchainBalance(partner, 1, { gasPrice: 1 });
+    await expect(tx).resolves.toEqual(txHash);
+  });
+
+  test('transferOnchainTokens', async () => {
+    const deps = makeDummyDependencies();
+    const raiden = new Raiden(
+      makeInitialState({ address, network, contractsInfo }),
+      deps,
+      combineRaidenEpics([initEpicMock]),
+      dummyReducer,
+    );
+    raiden.getTokenBalance = jest.fn(async () => BigNumber.from(1_000_000)) as any;
+
+    await raiden.start();
+    const tx = raiden.transferOnchainTokens(token, partner, 1);
```

Comment:
Same as above, but for tokens, no need to mind gasPrice then, but still check we're able to transfer the whole balance.